spec/factories/seeds/cohort_factory.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
instead of adding new factories like ECF (which has a mix of old and new world factories), could we reuse the existing factories everywhere? eg https://github.com/DFE-Digital/npq-registration/blob/main/spec/factories/cohorts.rb#L1

Same for the rest, what do you think?